src/swarm-scenario/Swarm/Game/Scenario/Topography/Placement.hs

```diff
@@ -82,10 +83,9 @@ data Placement = Placement
 
 instance FromJSON Placement where
   parseJSON = withObject "structure placement" $ \v -> do
-    sName <- v .: "src"
-    shouldTruncate <- v .:? "truncate" .!= True
-    p <-
-      Pose
-        <$> v .:? "offset" .!= origin
-        <*> v .:? "orient" .!= defaultOrientation
-    return $ Placement sName shouldTruncate p
+    src <- v .: "src"
+    truncateOverlay <- v .:? "truncate" .!= True
+    offset <- v .:? "offset" .!= origin
+    orient <- v .:? "orient" .!= defaultOrientation
```

Comment:
This makes it seem like `Pose` shoud've had a `FromJSON` instance 😄

Reply:
It probably should, feel free to add one!